### PR TITLE
[#575] Move EACL Table conversion in neofs package

### LINKED ISSUE #575: Move EACL Table conversion in neofs package
`ast` -> `eacl` and conversion and vice-versa is related to internal structure of NeoFS, so we can place it in `neofs` package.

### PR DESCRIPTION
**Pull request by masterSplinter01**
Closes #575

### CODE REVIEW

File: api/handler/neofs.go [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package handler
+
+import "github.com/nspcc-dev/neofs-sdk-go/eacl"
+
+type Ast struct {
+	Resources []*AstResource
+}
+
+type NeoFS interface {
```
Comment (KirillovDenis) [inline api/handler/neofs.go:9]: Isn't the `ACL` or `NeoFSACL` more appropriate name?

File: api/handler/acl.go [hunk 2]
```diff
@@ -604,89 +570,7 @@ func addPredefinedACP(acp *AccessControlPolicy, cannedACL string) (*AccessContro
...
-				resInfo.Version = filter.Value()
-			}
-		}
-	}
-
-	return resInfo
-}
-
-func mergeAst(parent, child *ast) (*ast, bool) {
+func MergeAst(parent, child *Ast) (*Ast, bool) {
```
Comment (KirillovDenis) [inline api/handler/acl.go:573]: Why is this function not moved to the `neofs` package?

**Comment by KirillovDenis** [comment]
Let's wait  a little for @alexvanin. I think we should reconsider the structure of acl logic.

**Comment by roman-khimov** [comment]
Obsolete.

**Pull request closed without merging**